reactioncommerce/reaction
# [WIP] Fix email verification url from the Getting Started email

Resolves #2623 

**How To Test**

1. Register as a new user with a valid email address.
2. Click on the `Get Started` in the email received.
3. Observe that it directs you to the verification page.
<img width="1280" alt="screen shot 2017-09-20 at 2 16 32 pm" src="https://user-images.githubusercontent.com/27197959/30641064-59b504e8-9e0e-11e7-8cff-3cac6af65667.png">
